pkgs/applications/networking/cluster/docker-machine/xhyve.nix

```diff
@@ -1,24 +1,34 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, Hypervisor, vmnet }:
+{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, cctools, Hypervisor, vmnet }:
 
 buildGoPackage rec {
   pname = "docker-machine-xhyve";
-  version = "0.3.3";
+  version = "0.4.0";
 
   goPackagePath = "github.com/zchee/docker-machine-driver-xhyve";
   goDeps = ./xhyve-deps.nix;
```

Comment:
would you mind removing `goDeps`, it should not be necessary given that upstream vendorizes its dependencies